async-nats/src/lib.rs

```diff
@@ -223,12 +223,13 @@ pub enum Command {
         respond: Option<String>,
     },
     Subscribe {
-        sid: u64,
+        id: u64,
         subject: String,
         queue_group: Option<String>,
+        sender: mpsc::Sender<Message>,
     },
     Unsubscribe {
-        uid: u64,
+        id: u64,
```

Comment:
`id` is effectivly `sid`, right? No reason to not call it `sid`.

Reply:
Did not re-implement remapping here, so yes.

Reply:
so lets just call it sid :)